graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -465,7 +497,9 @@ def __init__(
         # reverse_name_map contains all non-suppressed types, including those that were unchanged
         self.query_type = query_type
         self.scalar_types = frozenset(scalar_types)
-        self.builtin_types = frozenset({"String", "Int", "Float", "Boolean", "ID"})
+        self.builtin_types = frozenset(specified_scalar_types.keys())  # pylint: disable=E1101
```

Comment:
Please use the full name of the erroneous pass rather than the error code, because error codes are opaque and it's impossible to easily tell what is being suppressed.

If the full name is too long to fit on one line, use a `disable` on the line before, followed by an explicit `enable` on the next line.